src/main/webapp/pages/style.scss

```diff
@@ -1,8 +1,4 @@
-// @import '@splunk/themes/variables' as variable;
-@use '@splunk/themes/variables' as contentColorDefault;
-
 body {
-    // @include variables.contentColorDefault;
     min-width: 960px;
-    background-color: contentColorDefault;
+    background-color: #eee;
```

Comment:
Why are we adding it static?

Reply:
As discussed we are not sure about how to use Splunk UI's variable in a **.scss** file. Hence it's static. Once we get a solution to it we will change it later.